code/go/internal/validator/common_spec.go

```diff
@@ -7,12 +7,16 @@ package validator
 import (
 	"github.com/creasty/defaults"
 	"github.com/pkg/errors"
+
+	"github.com/elastic/package-spec/code/go/internal/spectypes"
 )
 
 type commonSpec struct {
-	AdditionalContents bool             `yaml:"additionalContents"`
-	Contents           []folderItemSpec `yaml:"contents"`
-	DevelopmentFolder  bool             `yaml:"developmentFolder"`
+	AdditionalContents bool               `yaml:"additionalContents"`
+	Contents           []folderItemSpec   `yaml:"contents"`
+	DevelopmentFolder  bool               `yaml:"developmentFolder"`
+	MaxSize            spectypes.FileSize `yaml:"maxSize"`
+	MaxContents        int                `yaml:"maxContents"`
```

Comment:
It looks like inconsistent: maxSize is spectype, but maxContents is int.

Reply:
`maxSize` is `spectypes.FileSize`, a kind of int.

Reply:
Yes, I'm wondering if we want to switch to spectypes, maybe MaxContents should be also a spectype.

Reply:
Well, I am adding spectypes for values that can benefit of special formatting or parsing, but a count of elements is ok as a plain int.